### PR TITLE
Optional TLS/Authentication for Contact Support Menu

### PR DESCRIPTION
**Pull request by dmccoystephenson**
# PR Details
### Problem
In the event that an email server does not support TLS or authentication, the Contact Support Menu ceases to function.

### Solution
TLS and authentication have been made optional via the introduction of the CSM_TLS_ENABLED and CSM_AUTH_ENABLED environment variables.

## How Has This Been Tested?
- Unit tests have been added and verified to be passing.
- Sending an email through Google's email server has been verified to work with TLS / AUTH enabled.
- Sending an email to a locally-running mail server has been verified to work with TLS / AUTH disabled.
- Sending an email through WYDOT's mail server has been verified to work with TLS / AUTH disabled.

## Types of changes
- [ ] Defect fix (non-breaking change that fixes an issue)
- [x] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that cause existing functionality to change)

## Checklist:
- [x] My changes require new environment variables:
  - [/] I have updated the docker-compose, K8s YAML, and all dependent deployment configuration files.
- [ ] My changes require updates to the documentation:
  - [ ] I have updated the documentation accordingly.
- [x] My changes require updates and/or additions to the unit tests:
  - [x] I have modified/added tests to cover my changes.
- [x] All existing tests pass.

### CODE REVIEW

File: services/api/src/contact_support.py [hunk 1]
```diff
@@ -31,25 +31,21 @@ class ContactSupportResource(Resource):
...
-        self.CSM_EMAIL_APP_USERNAME = os.environ.get("CSM_EMAIL_APP_USERNAME")
-        self.CSM_EMAIL_APP_PASSWORD = os.environ.get("CSM_EMAIL_APP_PASSWORD")
         self.CSM_EMAILS_TO_SEND_TO = os.environ.get("CSM_EMAILS_TO_SEND_TO")
         self.CSM_TARGET_SMTP_SERVER_ADDRESS = os.environ.get(
             "CSM_TARGET_SMTP_SERVER_ADDRESS"
         )
         self.CSM_TARGET_SMTP_SERVER_PORT = int(
             os.environ.get("CSM_TARGET_SMTP_SERVER_PORT")
         )
+        self.CSM_TLS_ENABLED = os.environ.get("CSM_TLS_ENABLED")
```
Comment (drewjj) [inline services/api/src/contact_support.py]: Can we actually default this to true even in the code so we don't have breaking changes the moment this is deployed in existing deployments without the addition of the new variables. (like CDOT's) I could foresee someone building the latest and deploying it without realizing new environment variables were added.

**Review by drewjj — CHANGES_REQUESTED** [review]
Just the one request, everything else seems great!

File: services/api/src/contact_support.py (hunk 1 above)
Comment (dmccoystephenson) [reply services/api/src/contact_support.py]: Good point! This has been addressed.

**Review by drewjj — APPROVED** [review]
Looks good!

**Pull request merged**